Formula/openvi.rb

```diff
@@ -0,0 +1,19 @@
+class Openvi < Formula
+  desc "Portable OpenBSD vi for UNIX systems"
+  homepage "https://github.com/johnsonjh/OpenVi#readme"
+  url "https://github.com/johnsonjh/OpenVi/archive/refs/tags/7.1.17.tar.gz"
+  sha256 "02974603f1cd3ba5727fd40f69f97925510d8f30c1ff3041adbc3e54e11f6d5f"
+  license "BSD-3-Clause"
+
+  uses_from_macos "ncurses"
+
+  def install
+    system "make", "install", "CHOWN=true", "LTO=1", "PREFIX=#{prefix}"
```

Comment:
What does `CHOWN` do here?

Reply:
`CHOWN=true` probably replaces every call to `chown` with `true`, making it do nothing.

Reply:
Yes - the `make install` phase will try to chown the executable during installation (to root).  Not necessary here.